pkg/ddevapp/config.go

```diff
@@ -139,6 +139,12 @@ func NewApp(appRoot string, includeOverrides bool) (*DdevApp, error) {
 		app.addUploadDir(uploadDirDeprecated)
 	}
 
+	// Remove dba
+	if nodeps.ArrayContainsString(app.OmitContainers, "dba") {
+		util.Warning("Container `dba` is no longer part of DDEV, please remove it from `omit_containers`")
```

Comment:
```suggestion
		util.Warning("PhpMyAdmin (`dba`) no longer part of DDEV core, please edit your `omit_containers` configuration to remove it")
```

Reply:
This definitely picks up both global and project omit_containers, right?

Reply:
No, this only takes the project config into account because I've misunderstood the last part of the issue description. But easy to fix, I just have to a second condition and we are good.

Reply:
Added with the commit below.